types/stylelint/index.d.mts

```diff
@@ -0,0 +1,46 @@
+export {
+	default,
+	BuiltInRules,
+	Config,
+	ConfigRuleSettings,
+	CosmiconfigResult,
+	CssSyntaxError,
+	CustomSyntax,
+	DisableOptions,
+	DisableOptionsReport,
+	DisablePropertyName,
+	DisableReportRange,
+	DisabledRange,
+	DisabledRangeObject,
+	DisabledWarning,
+	Formatter,
+	FormatterType,
+	Formatters,
+	GetLintSourceOptions,
+	GetPostcssOptions,
+	InternalApi,
+	LintResult,
+	LinterOptions,
+	LinterResult,
+	LonghandSubPropertiesOfShorthandProperties,
+	Plugin,
+	PostcssPluginOptions,
+	PostcssResult,
+	Problem,
+	PublicApi,
+	Rule,
+	RuleBase,
+	RuleContext,
+	RuleMessage,
+	RuleMessages,
+	RuleMeta,
+	RuleOptions,
+	RuleOptionsPossible,
+	RuleSeverity,
+	Severity,
+	ShorthandProperties,
+	StylelintPostcssResult,
+	Utils,
+	Warning,
+	WarningOptions,
```

Comment:
Unfortunately `export * from './index.js'` doesn’t work if the referenced module uses `export =`.